DnDDieRollCalculator/Program.cs

```diff
@@ -1,2 +1,21 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System;
+namespace RollReturn
+{
+    class Program
+    {
+        static int CalculateRoll(int dieValue)
+        {
+
+            Random random = new Random();
+            int roll = 0;
+
+            roll += random.Next(1, dieValue);
+
+            return roll;
```

Comment:
honestly, we're inevitably returning the value of the package's random.next(), which makes this function redundant.

Reply:
aka, wherever you intended to use CalculateRoll(), you can just call random.Next() directly